app/Http/Controllers/Admin/UserManagementController.php

```diff
@@ -32,17 +32,13 @@ class UserManagementController extends Controller
 	/**
 	 * Get the list of users for management purposes.
 	 *
-	 * @param ManagmentListUsersRequest $request
-	 * @param Spaces                    $spaces
-	 *
 	 * @return Collection<array-key, UserManagementResource>
 	 */
 	public function list(ManagmentListUsersRequest $request, Spaces $spaces): Collection
 	{
 		/** @var Collection<int,User> $users */
 		$users = User::select(['id', 'username', 'may_administrate', 'may_upload', 'may_edit_own_settings', 'quota_kb', 'description', 'note'])->orderBy('id', 'asc')->get();
 		$spaces_per_user = $spaces->getFullSpacePerUser();
-		/** @var Collection<int,array{0:User,1:array{id:int,username:string,size:int}}> $zipped */
```

Comment:
I would keep that one if possible, maybe remove the @var ? 
```suggestion
		/** Collection<int,array{0:User,1:array{id:int,username:string,size:int}}> $zipped */
```